spinta/backends/__init__.py

```diff
@@ -264,19 +264,6 @@ def simple_data_check(
             raise exceptions.FieldNotInResource(prop, property=key)
 
 
-@commands.simple_data_check.register(Context, DataItem, Ref, Property, Backend, object)
-def simple_data_check(
-    context: Context,
-    data: DataItem,
-    dtype: Ref,
-    prop: Property,
-    backend: Backend,
-    value: object,
-) -> None:
-    if value and not isinstance(value, dict):
-        raise exceptions.InvalidRefValue(prop, value=value)
```

Comment:
Why this check was deleted? And how this change is related to this task?

Reply:
replaced, going to check if all tests pass